From acc45031408a87d94b29b5f64aeddbf5cdded00e Mon Sep 17 00:00:00 2001
From: Your Name <you@example.com>
Date: Wed, 31 Jul 2019 23:46:38 +0000
Subject: [PATCH 2/5] fix build

---
 arch/openrisc/kernel/setup.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/openrisc/kernel/setup.c b/arch/openrisc/kernel/setup.c
index 3b57852b..e5c0bf38 100644
--- a/arch/openrisc/kernel/setup.c
+++ b/arch/openrisc/kernel/setup.c
@@ -39,6 +39,7 @@
 #include <linux/memblock.h>
 #include <linux/device.h>
 #include <linux/of_platform.h>
+#include <linux/screen_info.h>
 
 #include <asm/sections.h>
 #include <asm/segment.h>
@@ -51,6 +52,8 @@
 
 #include "vmlinux.h"
 
+struct screen_info screen_info;
+
 static unsigned long __init setup_memory(void)
 {
 	unsigned long bootmap_size;
-- 
2.11.0

